clients/web/src/stylesheets/body/userList.styl

```diff
@@ -58,3 +58,22 @@
     input[type="text"]
       width 170px
       margin-right 13px
+
+  .g-user-count-container
+    display inline-block
+    color #888
+    font-size 16px
+
+    i
+      position relative
+      top 2px
+
+    .g-user-count
+      position relative
+      background-color #eee
+      border-radius 3px
+      color rgba(0, 0, 0, 0.75)
+      bottom -7px
```

Comment:
Negative positioning always makes me a bit nervous. What you could do instead is:
* Make the parent element (`.g-user-count-container`) use `position relative`. That doesn't change anything directly, but [makes it considered to be a "positioned element"](https://developer.mozilla.org/en-US/docs/Web/CSS/position#Types_of_positioning).
* Remove the `position relative`,  `bottom -7px` and `right 7px` rules from `.g-user-count`.
* Add `position absolute`, `left 65%`, and `top 50%` rules to `.g-user-count`. Tweak the percentages to your liking.

This will result in `.g-user-count-container` having a more tightly constrained size, which will be exactly the same as the size of the `i` element within. The `.g-user-count` within will not longer affect the size of the `.g-user-count-container` in any way, but instead will just be positioned as a free element within that.